### PR TITLE
Add 3rd party license

### PR DESCRIPTION
**Pull request by kelseiv**
Closes #

_Describe your proposed changes here._

### CODE REVIEW

File: content/enterprise_influxdb/v1.9/about-the-project/contributing.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+description: Contribute to the InfluxDB OSS project.
+menu:
+  enterprise_influxdb_1_9_ref:
+    name: Contribute to InfluxDB OSS
+    weight: 30
+    parent: About the project
+---
+
+To learn how you can contribute to the InfluxDB OSS project, see
+[Contributing to InfluxDB](https://github.com/influxdata/influxdb/tree/1.8/CONTRIBUTING.md)
```
Comment (kelseiv) [inline content/enterprise_influxdb/v1.9/about-the-project/contributing.md:12]: Do we want to link to 2.0 OSS license info here too? If yes, merge after OSS 2.0.7 release.

File: content/enterprise_influxdb/v1.9/about-the-project/contributing.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+in the GitHub repository.
+
+Before contributing to the InfluxDB OSS project, you must complete and sign
+the [InfluxData Contributor License Agreement (CLA)](https://www.influxdata.com/legal/cla/),
+available on the InfluxData website.
+
+The [open source license for InfluxDB OSS](https://github.com/influxdata/influxdb/blob/master/LICENSE)
```
Comment (kelseiv) [inline content/enterprise_influxdb/v1.9/about-the-project/contributing.md:19]: Working with @psteinbachs to sort this out

File: content/enterprise_influxdb/v1.9/about-the-project/contributing.md (hunk 1 above)
Comment (timhallinflux) [reply content/enterprise_influxdb/v1.9/about-the-project/contributing.md:12]: not required. Enterprise is licensed commercially.

Comment (timhallinflux) [reply content/enterprise_influxdb/v1.9/about-the-project/contributing.md:12]: however, I think this page should be dropped.  Intrepid developers could try and contribute to InfluxDB Enterprise by contributing code to OSS....but, let's just drop this page entirely.

File: content/enterprise_influxdb/v1.9/about-the-project/third-party.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+| [pkcs7](https://github.com/fullsailor/pkcs7) | [MIT](https://opensource.org/licenses/MIT) | Statically linked |
+| [pretty](https://github.com/kr/pretty) | [MIT](https://opensource.org/licenses/MIT) | Statically linked |Statically linked|
+| [raft](https://github.com/hashicorp/raft) | [Mozilla Public License 2.0](https://opensource.org/licenses/MPL-2.0)  | Statically linked |Statically linked|
+| [raft-boltdb](https://github.com/hashicorp/raft-boltdb) | [Mozilla Public License 2.0](https://opensource.org/licenses/MPL-2.0)  | Statically linked |Statically linked|
+| [sqlx](https://github.com/jmoiron/sqlxhttps://github.com/jmoiron/sqlx) | [MIT](https://opensource.org/licenses/MIT) | Statically linked |Statically linked|
+| [text](https://github.com/kr/text) | [MIT](https://opensource.org/licenses/MIT) | Statically linked |Statically linked|
+| [yamux](https://github.com/hashicorp/yamux/)| [Mozilla Public License 2.0](https://opensource.org/licenses/MPL-2.0) | Statically linked |Statically linked|
+
+\* BSD 3-Clause is also known as the "New" or "Modified" BSD License. BSD 2-Clause is also known as the "Simplified" or "Free" BSD License.
```
Comment (timhallinflux) [inline content/enterprise_influxdb/v1.9/about-the-project/third-party.md:49]: line 49 looks like a stray?

**Review by timhallinflux — CHANGES_REQUESTED** [review]

File: content/enterprise_influxdb/v1.9/about-the-project/third-party.md (hunk 3 above)
Comment (kelseiv) [reply content/enterprise_influxdb/v1.9/about-the-project/third-party.md]: Remove note about BSD licenses, now we're linked to the license.

**Comment by kelseiv** [comment]
Merging third party license content into Enterprise 1.9 branch

**Pull request merged**